### PR TITLE
Issue/#18  create form component

### LINKED ISSUE #18: Create Form Component
@MaksymTaratuta55 

We need to create a `Form` component.  Basically, it will be a wrapper for the Form Input components (`Input`, `Select`, etc.) with some starting `grid` classes.  We would use it like this:

```
<Form>
  <Input />
  <TextArea />
  <Select>...</Select>
</Form>
```

In terms of the starting css classes -- this is the default:  `grid grid-cols-1 gap-y-6 gap-x-4 sm:grid-cols-6`

### PR DESCRIPTION
**Pull request by MaksymTaratuta55**
closes #18

### CODE REVIEW

**Comment by MaksymTaratuta55** [comment]
Should be closed after #16

File: src/components/form/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import theme from "@/themes/theme";
+
+const Textarea = ({ className, children, id, name, onSubmit, onReset }) => (
```
Comment (thecodeclimber) [inline src/components/form/Form.jsx]: The component should be named `Form`, not `Textarea`.

File: src/components/form/Form.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  <form
+    css={[theme.form.base, className]}
+    id={id}
```
Comment (thecodeclimber) [inline src/components/form/Form.jsx:7]: Do we need `id`?  Can we not just set `id` to whatever `name` is set to?  Or, alternatively, let's set the default value of `id` to whatever `name` is set to and only override it if `id` is explicitly set.  

That reminds me -- are any of these props required?  If so, should we not indicate that in propTypes?

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/components/form/Form.jsx (hunk 1 above)
Comment (MaksymTaratuta55) [reply src/components/form/Form.jsx]: yeah 😅, sorry

**Comment by MaksymTaratuta55** [comment]
@thecodeclimber, updated

**Comment by thecodeclimber** [comment]
@MaksymTaratuta55 

Can you please resolve the conflicts with this pull request?

**Comment by MaksymTaratuta55** [comment]
@thecodeclimber, sure

**Comment by MaksymTaratuta55** [comment]
@thecodeclimber, updated

File: src/components/form/Form.jsx (hunk 1 above)
Comment (thecodeclimber) [reply src/components/form/Form.jsx]: No worries :)

**Pull request merged**